src/components/enterprise/App.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Helmet } from 'react-helmet';
+import { Router } from '@reach/router';
+import { ConnectedProfilePage } from  '@edx/frontend-app-profile';
+
+import { configuration } from '../../../environment';
+import { withAuthentication } from '../common/with-authentication';
+import { Layout } from '../common/layout';
+import { DashboardPage } from './dashboard';
+
+const App = (props) => {
+  const { pageContext } = props;
+  const { enterpriseName } = pageContext;
+  return (
+    <>
+      <Helmet title={enterpriseName} />
+      <Router>
+        <Layout path="/" pageContext={pageContext}>
+          <DashboardPage path="/" pageContext={pageContext} />
```

Comment:
If we're already passing `pageContext` into `Layout`, `pageContext` data should be accessible via the context API within `DashboardPage` and probably don't need to pass it as a prop here.